test/integration/053_custom_materialization/test_custom_materialization.py

```diff
@@ -98,3 +98,19 @@ def test_postgres_default_dependency(self):
         self.run_dbt(['deps'])
         # this should error because the override is buggy
         self.run_dbt(['run'], expect_pass=False)
+
+
+class TestOverrideDefaultReturn(BaseTestCustomMaterialization):
+    @property
+    def project_config(self):
+        return {
+            'config-version': 2,
+            'macro-paths': ['override-view-return-no-relation']
+        }
+
+    @use_profile('postgres')
+    def test_postgres_default_dependency(self):
+        self.run_dbt(['deps'])
+        self.run_dbt(['run'], expect_pass=False)
+
```

Comment:
If you want, you can check the results for the explicit error message you're expecting:
```python
        results = self.run_dbt(['run'], expect_pass=False)
        assert 'did not explicitly return a list of relations' in results[0].message
```
I wouldn't say that's required here, though it could be a nice way to ensure we're providing users with a helpful error message, and not just failing silently.